Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -680,3 +680,90 @@ lemma not_isCyclic_iff_exponent_eq_prime [Group α] {p : ℕ} (hp : p.Prime)
   · exact False.elim <| h_cyc <| isCyclic_of_orderOf_eq_card g <| by aesop
 
 end Exponent
+
+section ZMod
+
+open Subgroup AddSubgroup
+
+variable {G H : Type*}
+
+/-- The isomorphism between `(Multiplicative G) / ⟨.ofAdd g⟩` and `Multiplicative (G / ⟨g⟩)`. -/
+def multiplicativeQuotientZpowersZmultiplesMulEquiv [AddCommGroup G] (g : G) :
+    Multiplicative G ⧸ zpowers (Multiplicative.ofAdd g) ≃* Multiplicative (G ⧸ zmultiples g) := by
+  rfl
```

Comment:
since this is data:
```suggestion
    Multiplicative G ⧸ zpowers (Multiplicative.ofAdd g) ≃* Multiplicative (G ⧸ zmultiples g) :=
  MulEquiv.refl _
```

Please also add a lemma that says what `multiplicativeQuotientZpowersZmultiplesMulEquiv` applied to `QuotientGroup.mk` is, and similarly for `.symm`.